index.js

```diff
@@ -834,7 +834,7 @@ function buildConstSerializer (location, input) {
     `
   }
 
-  code += `json += '${JSON.stringify(schema.const)}'`
+  code += `json += '${JSON.stringify(schema.const).replaceAll("'", "\\'")}'`
```

Comment:
```suggestion
  code += `json += '${JSON.stringify(schema.const).replace(SINGLE_TICK "\\'")}'`
```

and on top of the this file:

```js
const SINGLE_TICK = /'/g
```